src/ucp/wireup/select.c

```diff
@@ -420,7 +425,8 @@ ucp_wireup_add_memaccess_lanes(ucp_ep_h ep, unsigned address_count,
                                ucp_wireup_lane_desc_t *lane_descs,
                                ucp_lane_index_t *num_lanes_p,
                                const ucp_wireup_criteria_t *criteria,
-                               uint64_t tl_bitmap, uint32_t usage, int mandatory)
+                               uint64_t tl_bitmap, uint32_t usage,
+                               ucp_select_memaccess_lanes_t mode)
```

Comment:
maybe just always use "all" mode, and later on pick best N?

Reply:
for now RMA & atomics will use only one lane (and ucp_wireup_select_transport selects one best), after migration to multi-lane mode we may switch it to "all" mode

Reply:
how would multi-lane mode select only the best 3 (since MAX_OPS=3)?

Reply:
isn't it limitation for MD count?
on RNDV memory registration it should select 3 memory domains where lanes have best score

Reply:
ok, i see, so your intention is that the best lanes would be selected during rndv protocol? makes sense.. then would make the argument to the function "boolean"(int) instead of enum since there are only going to be 2 possible options..

Reply:
enum allows to set human-readable value instead of just 1/0

ok, will replace